taxi/views.py

```diff
@@ -25,7 +29,27 @@
         "num_visits": num_visits + 1,
     }
 
-    return render(request, "taxi/index.html", context=context)
+    return render(request, "taxi/index.html", context)
+
+
+class DriverCreateView(CreateView):
+    model = User
+    form_class = DriverCreationForm
+    template_name = "taxi/driver_form.html"
+    success_url = reverse_lazy("taxi:driver-list")
```

Comment:
Ensure that the URL name uses hyphens instead of underscores as per the checklist. It should be `driver-list` instead of `driver-list`.